src/passes.cc

```diff
@@ -293,6 +294,12 @@ void ReportStatisticsPass::runOnProgram(ql::quantum_program *program)
     ql::report_statistics(program, program->platform, "todo-inout", getPassName(), "# ");
 }
 
+void VisualizerPass::runOnProgram(ql::quantum_program *program)
+{
+    Layout layout;
```

Comment:
where is this initialized?

Reply:
It is initialized with default values when you create a Layout object. In the future the default values will instead be read from a configuration file, and can also be changed by including a Layout object in the pass options.